src/main/resources/db/migration/V2_4__alter_column_melding_type.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE MELDING ALTER COLUMN TYPE TYPE VARCHAR(50);
```

Comment:
Øker størrelsen på denne for å få plass til `FORESPORSEL_PASIENT_PAMINNELSE`

Reply:
Men ble det riktig med "TYPE TYPE"?

Reply:
Ja, tror det. Blir dobbel type siden kolonnen heter type 😅

Reply:
Kolonnenavnet kunne evt vært i små bokstaver, pleier vi ikke det? Og så er SQL-keywords i all-capslock